src/main/java/org/sagebionetworks/bridge/services/StudyActivityEventService.java

```diff
@@ -212,11 +237,12 @@ private void addIfPresent(List<StudyActivityEvent> events, Map<String, DateTime>
             events.add(event);
         }
     }
-
+    
     /**
-     * If the triggering event is mutable, these events can be created as well.
+     * If the triggering event is mutable, these events can be created as well. Returns true
+     * if any study burst failed to update, or false otherwise.
```

Comment:
The comment addition might not be necessary since it doesn't return a boolean. Could change it to say it adds the failures to the given list, but you also covered that in the comment on 133.